Firestore/Example/Tests/Core/FSTQueryListenerTests.mm

```diff
@@ -48,9 +49,31 @@
 using firebase::firestore::util::ExecutorLibdispatch;
 using firebase::firestore::util::Status;
 using firebase::firestore::util::StatusOr;
+using testing::ElementsAre;
+using testing::IsEmpty;
 
 NS_ASSUME_NONNULL_BEGIN
 
+namespace {
+
+/**
+ * Returns a copy of the given snapshot, with excludesMetadataChanges set to true.
+ */
+ViewSnapshot ExcludingMetadataChanges(const ViewSnapshot &snapshot) {
+  return ViewSnapshot{
+      snapshot.query(),
+      snapshot.documents(),
+      snapshot.old_documents(),
+      snapshot.document_changes(),
+      snapshot.mutated_keys(),
+      snapshot.from_cache(),
+      snapshot.sync_state_changed(),
+      true,
```

Comment:
I'd rework this slightly to:
a) `/* excludesMetadataChanges = */ true,` on line 71.
b) drop the docstring.

Both optional.

Reply:
Done and done.